apps/console/src/features/users/components/add-user.tsx

```diff
@@ -97,6 +105,23 @@ export const AddUser: React.FunctionComponent<AddUserProps> = (props: AddUserPro
     const USERNAME_HAS_INVALID_CHARS_ERROR_MESSAGE: string = t("console:manage.features.user.forms.addUserForm." +
         "inputs.username.validations.invalidCharacters");
 
+    useEffect(() => {
+
```

Comment:
```suggestion
```

Reply:
Resolved in [258cafe](https://github.com/wso2/identity-apps/pull/4652#:~:text=address%20comments-,258cafe,-Merge%20state)